concourse/steps/component_descriptor.py

```diff
@@ -38,6 +38,7 @@ def dump_component_descriptor_v2(component_descriptor_v2: cm.ComponentDescriptor
 def base_component_descriptor_v2(
     component_name_v2: str,
     effective_version: str,
+    source_labels: tuple,
```

Comment:
please use proper type-hint. we do not require a tuple here.

-> typing.List[cm.Label] (please stick w/ the (new) convention of aliased importing (import gci.componentmodel as cm)